### ISSUE TITLE
Suggest question mode in chat

### ISSUE THREAD

**Issue by rossng** [clowdr-app/clowdr#251]
When the user adds a question mark to their chat message, we should suggest that they switch to question mode.

**Comment by bcpierce00** [comment]
This feature could be very annoying if we don't get the UX right!

What about showing a "hint" dialog the *first* time someone posts a message
ending in a ? in a chat channel associated with a streaming event (not just
any event) explaining how to create questions?  (There are probably other
places where such one-time hints could be useful...  E.g., we could use
them to give people a quick tour of Clowdr, when they first log in or at
some later point when they choose to take a tour...)

**Comment by rossng** [comment]
Yes, that is probably a good idea. In any case, it definitely shouldn't be a modal/blocking thing.
